src/client/cli/commands/publish.js

```diff
@@ -14,10 +14,11 @@ type HandlerOptions = {
   apiUrl: string,
   idSelector: string,
   contentSelector: ?string,
```

Comment:
Oh yeah, I kind of got those mixed up when first defining this.  I think it makes more sense to have `field?: type`, since that's really what we're saying; the option might not be present.  They both have the same effect when you use them in that you have to check for `undefined`.